src/app/api/routes/sites.py

```diff
@@ -24,6 +24,17 @@ async def create_site(payload: SiteIn, _=Security(get_current_access, scopes=["a
     return await crud.create_entry(sites, payload)
 
 
+@router.post("/no-alert/", response_model=SiteOut, status_code=201, summary="Create a new no-alert site")
```

Comment:
/no-alert/ is confusing I think.

create-no-alert-site or something like that would be better IMHO

Reply:
I disagree, having route names that are too long isn't a good idea.
Regarding the naming, I thought that somehow that do a POST method on sites/no-alert/ would easily understand?
(the naming on the route should be independent on the method type, so adding "create" in the name sounds like a bad idea to me :man_shrugging: )

Reply:
Fair enough.